src/modules/ce/components/ReferralPage.tsx

```diff
@@ -57,34 +53,39 @@ const ReferralPage: React.FC<Props> = ({}) => {
     [referral]
   );
 
-  const currentPath = useCurrentPath();
+  const isMobile = useIsMobile();
 
   if (loading) return <Loading />;
   if (error) throw error;
   if (!referral) return <NotFound />;
 
   return (
     <>
-      <CommonStickyBar height={REFERRAL_HEADER_HEIGHT}>
-        <Stack direction='column' gap={2}>
+      {/* Apply 0 padding to the bar, so the divider can take up full height */}
+      <CommonStickyBar sx={{ py: 0 }}>
+        <Stack
+          divider={
+            <Divider
+              orientation={isMobile ? 'horizontal' : 'vertical'}
+              flexItem
+            />
+          }
+          direction={isMobile ? 'column' : 'row'}
+          alignItems={isMobile ? '' : 'center'}
+          gap={2}
+        >
           <Stack
+            sx={{ py: 2 }}
+            flex={1}
             direction='row'
             alignItems='center'
             justifyContent='space-between'
           >
-            <Typography variant='h3' component='h1' display='inline-block'>
-              Referral to {referral.opportunity.name}
-            </Typography>
-            <ReferralStatusChip status={referral.status} />
-          </Stack>
-          <Stack
-            direction='row'
-            gap={2}
-            divider={<Divider orientation='vertical' flexItem />}
-          >
-            <Stack direction='row' alignItems='center' gap={0.5}>
-              <Person sx={{ color: 'grayscale.main' }} />
-              <Typography variant='body1'>
+            <Box>
+              <Typography variant='overline'>
+                Referral to {referral.opportunity.name}
+              </Typography>
+              <Typography variant='h3' component='h1'>
```

Comment:
We're waiting for input from design on this question of where to display the client name. Here's how I've implemented it for now:

![Screenshot 2025-05-22 at 2 59 26 PM](https://github.com/user-attachments/assets/4493ef20-902a-4782-a3dc-e6ecb8b9f579)

Reply:
Sounds good. My thoughts are:
- client name shouldn't be linked. (client may have multiple records; client may not be in hmis; user may have access to see their name but not their profile; user may accidentally click away from current page; etc). we can add links to hmis/warehouse dashboards to the client detail  panel
- I think it adds too much height/padding to the top bar.. I think thats the nature of using overline text though, maybe only solvable by having one horizontal title
- overline text color doesn't match our usual approach in `PageTitle` where overline text is the dark primary